.github/infrastructure/conformance/azure/setup-azure-conf-test.sh

```diff
@@ -228,7 +228,7 @@ STORAGE_CONTAINER_VAR_NAME="AzureBlobStorageContainer"
 STORAGE_QUEUE_VAR_NAME="AzureBlobStorageQueue"
 
 # Derived variables
-ADMIN_ID="$(az ad user list --filter "userPrincipalName eq '${ADMIN_UPN}'" --query "[].objectId" --output tsv)"
```

Comment:
Reviewers: the script was not working with the updated Azure SDK

Reply:
I can confirm this is necessary.